kotlin-analysis/build.gradle.kts

```diff
@@ -15,3 +15,7 @@ dependencies {
 registerDokkaArtifactPublication("dokkaAnalysis") {
     artifactId = "dokka-analysis"
 }
+
+binaryCompatibilityValidator {
+    enabled.set(false)
+}
```

Comment:
the kotlin-analysis projects have API verification disabled, but perhaps they should indeed be validated? After all, they are published artifacts.